APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -114,7 +114,7 @@ - (void)viewDidLoad
     APCUser  *user = appDelegate.dataSubstrate.currentUser;
     self.demographicUploader = [[APCDemographicUploader alloc] initWithUser:user];
     
-    
+    [self refreshView];
```

Comment:
@EricSiegNW 
I'm guessing that the refresh is called in viewWillAppear in the case where this is a return from showing something that could change the profile (either push on or modally).  Ideally, it would only get conditionally called if not a first appearance, but since that adds a level of state tracking, not really that much of a performance hit to just reload. *shrug*